extensions/credentials/identity-hub-credentials-jwt/src/main/java/org/eclipse/edc/identityhub/credentials/jwt/JwtPresentation.java

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
Please add a unit-test for serialization/deserialization

Reply:
I added that and indeed one check for a mandatory field was not working properly